_typos.toml

```
@@ -7,4 +7,4 @@
 ba="ba" # 7zコマンドの-baオプション
 
 [files]
-extend-exclude = ["package-lock.json", "src/store/project.ts", "*.svg"]
+extend-exclude = ["package-lock.json", "src/store/project.ts", "*.svg", "tests/unit/dummyImage.ts"]
```

さすがにtomlに書かないといけないのは避けていきたいかなと！
```suggestion
extend-exclude = ["package-lock.json", "src/store/project.ts", "*.svg"]
```